app/src/main/java/com/tugraz/quizlet/frontend/LoginActivity.kt

```diff
@@ -6,19 +6,31 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.Button
 import android.widget.EditText
+import android.widget.TextView
 import android.widget.Toast
 import com.tugraz.quizlet.R
 import com.tugraz.quizlet.frontend.StartActivity
 import io.realm.mongodb.AppException
+import org.intellij.lang.annotations.Language
 import java.util.*
 
 
 class LoginActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-
         setContentView(R.layout.activity_login)
 
+        // TODO: Set new Text for lang support
```

Comment:
is this still needed?

Reply:
Yes, for later changes on second release.